doc/source/API/SetupTemplates.rst

```diff
@@ -0,0 +1,176 @@
+Example
+
+.. code:: python
+
+    Launch AEDT 2021 R1 in Non-Graphical mode
+
+    from pyaedt import Hfss
+
+    hfss = Hfss()
+    # Any property of this setup can be found in this page.
+    hfss.props["AdaptMultipleFreqs"] = True
+    hfss.update()
+
+
+
+
+
+HFSS Templates
+==============
+
+Hfss Automatic Setup
+--------------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.HFSSDrivenAuto
+
+Hfss Driven Modal
+-----------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.HFSSDrivenDefault
+
+Hfss Eigen Mode
+---------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.HFSSEigen
+
+Hfss Transient
+--------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.HFSSTransient
+
+
+Hfss SBR+
+---------
+
+.. autodata:: pyaedt.modules.SetupTemplates.HFSSSBR
+
+
+
+Maxwell Templates
+=================
+
+Maxwell Transient
+-----------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.MaxwellTransient
+
+
+Maxwell Magnetostatic
+---------------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.Magnetostatic
+
+
+Maxwell Electrostatic
+---------------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.Electrostatic
+
+Maxwell EddyCurrent
+-------------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.EddyCurrent
+
+Maxwell ElectricTransient
+-------------------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.ElectricTransient
+
+
+Q3D Templates
+=============
+
+Q3d  Analysis
+-------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.Matrix
+
+
+Q2d Close Analysis
+------------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.Close
+
+Q2d Open Analysis
+-----------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.Open
+Icepak Templates
+================
+
+
+Transient Flow Only
+-------------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.TransientFlowOnly
+
+Temperature Flow Only
+---------------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.TransientTemperatureOnly
+
+
+Transient & Temperature Flow
+----------------------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.TransientTemperatureAndFlow
+
+
+Nexxim Analsyis
+===============
+
+LNA Analysis
+------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.NexximLNA
+
+
+DC Analysis
+-----------
+
+.. autodata:: pyaedt.modules.SetupTemplates.NexximDC
+
+
+Transient Analysis
+------------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.NexximTransient
+
+
+HFSS 3D Layout
+==============
+
+.. autodata:: pyaedt.modules.SetupTemplates.HFSS3DLayout
+
+
+Mechanical Analysis
+===================
+
+
+Mechanical Thermal
+------------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.MechTerm
+
+
+Mechanical Modal
+----------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.MechModal
+
+
+Mechanical MechStructural
+-------------------------
+
+.. autodata:: pyaedt.modules.SetupTemplates.MechStructural
+
+
+RMXPrt Analysis
+===============
+
+
+
+.. autodata:: pyaedt.modules.SetupTemplates.GRM
```

Comment:
```suggestion
.. currentmodule:: pyaedt.modules.SetupTemplates

HFSS Templates
==============

.. autosummary::
   :toctree: _autosummary

   HFSSDrivenAuto
   HFSSDrivenDefault
   HFSSEigen
   HFSSTransient
   HFSSSBR


Maxwell Templates
=================

.. autosummary::
   :toctree: _autosummary

   MaxwellTransient
   Magnetostatic
   Electrostatic
   EddyCurrent
   ElectricTransient


Q3D Templates
=============

.. autosummary::
   :toctree: _autosummary

   Matrix
   Close
   Open

Icepak Templates
================

.. autosummary::
   :toctree: _autosummary

   TransientFlowOnly
   TransientTemperatureOnly
   TransientTemperatureAndFlow


Nexxim Analsyis
===============

.. autosummary::
   :toctree: _autosummary

   NexximLNA
   NexximDC
   NexximTransient


HFSS 3D Layout
==============

.. autosummary::
   :toctree: _autosummary

   HFSS3DLayout


Mechanical Analysis
===================

.. autosummary::
   :toctree: _autosummary

   MechTerm
   MechModal
   MechStructural


RMXPrt Analysis
===============

.. autosummary::
   :toctree: _autosummary

   GRM

```

This renders way better:
![tmp](https://user-images.githubusercontent.com/11981631/126371535-1679eb1e-4beb-48e7-8e9d-43734d97ed25.png)